big5/workload.json

```diff
@@ -15,9 +15,11 @@
       "base-url": "https://opensearch-benchmark-workloads.s3.amazonaws.com/corpora/big5",
       "documents": [
         {
-          "source-file": "documents.json",
-          "document-count": 69223950,
-          "uncompressed-bytes": 64048001338
+          "source-url": "{{ document_url | safe }}",
+          "source-file": "{{ document_file | default('documents.json') }}",
+          "document-count": {{ document_count }},
```

Comment:
How are these parameters filled in? Thought we needed the hard coded document-count, compressed bytes, and uncompressed bytes still?

Reply:
Correct.  The document count is still needed, however since the corpus is not supplied yet, a user will need to generate their own and pass in the count using `--workload-params`. Once an appropriate data corpus becomes available, that will be changed to a default value.

The `source-url` field is new and optional.  It requires some changes to the code-base and permits access to a source file via the full path to it, which may not conform to the standard convention.